app/assets/javascripts/discourse/app/templates/preferences/interface.hbs

```diff
@@ -117,6 +117,7 @@
     <ComboBox @valueProperty="value" @content={{this.titleCountModes}} @value={{this.model.user_option.title_count_mode}} @id="user-title-count-mode" @onChange={{action (mut this.model.user_option.title_count_mode)}} />
   </div>
   <PreferenceCheckbox @labelKey="user.skip_new_user_tips.description" @checked={{this.model.user_option.skip_new_user_tips}} @class="pref-new-user-tips" />
+  <a href {{on "click" (action "resetSeenPopups")}}>{{i18n "user.reset_seen_popups"}}</a>
```

Comment:
I don't think this is a proper use of a link tag here. I'm curious if there is a reason we chose a link tag here over a button tag.

Reply:
I just think that visually it looks better. I can make it a button.

Reply:
I think we have a btn-flat utility class which you can use to make the button look like a link.

Reply:
It does not look 100% like a link and does not feel like one either. I'll use that instead anyway.

Reply:
hmm I don't want to over complicate things here so let's just leave this as it is. Ideally, we would not be using a link meant mainly for linking pages and not really for triggering actions. Maybe we need a different button class for cases like this.